src/database/DatabaseConnectionHandler.java

```diff
@@ -13,7 +13,7 @@
 
   public static Connection getConnection() {
     try {
-      connection = DriverManager.getConnection(ORACLE_URL, "USER", "PASSWORD");
+      connection = DriverManager.getConnection(ORACLE_URL, "ora_wrigh63", "a10123248");
```

Comment:
remove this if you don't want your credentials online, leave it if you don't care